src/Options.js

```diff
@@ -164,7 +165,7 @@ export class Options {
   }
 
   get sourceMaps() {
-    return this.sourceMaps_;
+    return this.lowResolutionSourceMap || this.sourceMaps_;
```

Comment:
As you probably know, the sourcemaps option controls the output format, inline, memory, or file. I suppose we could have the same values on lowResolutionSourceMap but I'm not keen on more tricky options code.